src/HashBus.Twitter.Monitor.CatchUp/Program.cs

```diff
@@ -43,15 +52,15 @@ static void Main()
 
             Console.Title = typeof(Program).Assembly.GetName().Name;
 
-            App.Run(
+            await App.Run(
                 maximumNumberOfTweetsPerCatchUp,
                 defaultTransactionTimeout,
                 nserviceBusConnectionString, 
                 endpointName,
                 consumerKey, 
                 consumerSecret,
                 accessToken,
-                accessTokenSecret);
+                accessTokenSecret, monitorCatchUpAddress, applicationAddress, monitorAddress);

```

Comment:
Please put each argument on it's own line, as per the rest of the argument list.